net-proxy/mitmproxy/mitmproxy-5.1.1.ebuild

```diff
@@ -54,6 +57,9 @@ python_prepare_all() {
 	# loosen dependencies
 	sed -i '/>/s/>.*/",/g' setup.py || die
 
+	# fix Brotli dependency
+	sed -i 's/Brotli/brotlipy/g' setup.py || die
```

Comment:
Depend on app-arch/brotli[python,${PYTHON_USEDEP}] instead.

Reply:
Thanks, yes that did the trick.  I was racking my head on that one for a minute.

Reply:
If it works with brotlipy, it's better to use that. The hyper-h2 suite requires it, and the two block each other, so a lot of people won't be able to install this if it needs `app-arch/brotli[python]`.

Reply:
It does work, just tested with Python 3.8 and it seems to function just fine. Repoman still has a lot of complaints.